README.md

```diff
@@ -63,6 +61,40 @@ module "spacelift" {
 }
 ```
 
+### Use an existing EKS cluster
+
+If you only want to deploy the Spacelift-specific infrastructure, and have an existing EKS cluster that you want to deploy Spacelift to, you may want to take a look at our [terraform-aws-spacelift-selfhosted](https://github.com/spacelift-io/terraform-aws-spacelift-selfhosted) and [terraform-aws-iam-spacelift-selfhosted](https://github.com/spacelift-io/terraform-aws-iam-spacelift-selfhosted) modules.
+
+See below for an example.
+
+```hcl
+
+module "spacelift" {
+  source = "github.com/spacelift-io/terraform-aws-spacelift-selfhosted?ref=v1.3.1"
+  # ...
```

Comment:
don't we need to add create_vpc = false here?

Reply:
ah nvm, we need the VPC because of the RDs

Reply:
That's not really related to this module, here this is just a small example about how you should configure things if you don't want to use this EKS module. I just wanted to point out that you need to configure `kubernetes_role_assumption_config.oidc_provider` and also add a `aws_vpc_security_group_ingress_rule.cluster_database_ingress_rule`

Reply:
👌